FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +23,45 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case .failure(_):
```

Comment:
Since the specific error instance is being ignored, we can remove the attribution completely:
```suggestion
			case .failure:
```